host/src/pack/dpkg/etc/rc.civ.post

```diff
@@ -65,6 +65,6 @@ balloon_civ
 
 /opt/lg/bin/LG_B1_Client_clipboard guestClipboard:enable=true &
 
-pause_civ
+flock -n -x /var/lock/civ_startapp.lock pause_civ ||
```

Comment:
why need ||

Reply:
if flock failed to acquire the lock, the whole service will be stopped. So I add a "||" to ignore the failure here